custom_item_loader/src/Main.cs

```diff
@@ -18,11 +18,13 @@ private static bool Load(UnityModManager.ModEntry modEntry)
 			try
 			{
 				MyModEntry = modEntry;
-				
+
 				modEntry.OnUnload = OnUnload;
 
 				myHarmony = new Harmony(MyModEntry.Info.Id);
 				myHarmony.PatchAll(Assembly.GetExecutingAssembly());
+				// force load of custom item components
+				modEntry.Logger.Log($"Loading required assembly for {typeof(custom_item_components.GadgetItem)}");
```

Comment:
What is this? This log somehow loads a DLL?

Reply:
It's the best option I've found to force load of that dll.  Unity and UMM don't load anything but the main dll.  You have to either directly reference a type in the DLL, or do other even more convoluted things.  The log line prevents the line from being optimized away.

The DLL would load eventually, but not before it's needed to handle loading of prefabs which have that DLL in them.